Buildings/Controls/OBC/FDE/DOAS/EconMode.mo

```diff
@@ -0,0 +1,127 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block EconMode "This block calculates when economizer mode is active."
+  parameter Real econCooAdj(
+  final unit = "K",
+  final displayUnit = "degC",
+  final quantity = "ThermodynamicTemperature") = 2
+  "Value subtracted from supply air temperature cooling set point.";
+
+  parameter Real delayTimeEcoMod(
+  final unit= "s",
+  final quantity="Time")=10
+  "Delay added to compensate for CDL not processing latch correctly.";
+
+
+
+  // ---inputs---
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uFanSupPro
+    "True when supply fan is proven on" annotation (Placement(
+      visible=true,
+      transformation(
+        origin={2,-40},
+        extent={{-142,42},{-102,82}},
+        rotation=0),
+      iconTransformation(
+        origin={0,0},
+        extent={{-142,50},{-102,90}},
+        rotation=0)));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput TAirOut
+    "Outside air temperature" annotation (Placement(
+      visible=true,
+      transformation(
+        origin={2,-38},
+        extent={{-142,6},{-102,46}},
+        rotation=0),
+      iconTransformation(
+        origin={0,0},
+        extent={{-142,-20},{-102,20}},
+        rotation=0)));
+
+  // ---outputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yEcoMod "True when economizer mode is active."
+  annotation (
+    Placement(transformation(extent = {{104, -20}, {144, 20}}), iconTransformation(extent = {{102, -20}, {142, 20}})));
+
```

Comment:
Arrange input and output interfaces as per OBC convention. Add protected keyword after all interfaces have been declared